ec2utils/ec2publishimg/Makefile

```diff
@@ -0,0 +1,25 @@
+DESTDIR=
+MANPATH=/usr/share/man
+PREFIX=/usr
+NAME=ec2publishimg
+dirs = lib man
+files = Makefile README.md LICENSE ec2publishimg setup.py
+
+nv = $(shell rpm -q --specfile --qf '%{NAME}-%{VERSION}\n' *.spec)
+verSpec = $(shell rpm -q --specfile --qf '%{VERSION}' *.spec)
+verSrc = $(shell cat lib/ec2utils/publish_VERSION)
+ifneq "$(verSpec)" "$(verSrc)"
```

Comment:
This isn't specific to the code in this PR, just a general comment.

I *hate* having the duplicate version numbers. Can we come up with a solution that only requires maintaining a single entry point for the version number?